cpu/samd21/cpu.c

```diff
@@ -204,8 +204,18 @@ static void clk_init(void)
                          | SYSCTRL_XOSC32K_ENABLE;
 #endif
 
+    /* Setup Clock generator 3 with divider 4 (1024 Hz) */
```

Comment:
Is it 1024Hz? Or is it 1000Hz? Also, I would rather see "divide by 32" in the comment.

Reply:
32768 Hz / 32 = 1024 Hz
I will update the comment.

Reply:
And use GCLK3 in the comment so that it is easier to find all GCLK's in the future.